### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by Lamperoyge**
## :pushpin: References

- **Wonder issue:**
- **Related pull-requests:**

## :movie_camera: Screenshot or Video

https://www.loom.com/share/10feb9afd91e4ae694ec6ee87547cc15

## :cake: Checklist:

- [ ] I self-reviewed my changes
- [ ] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [ ] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-bot-admin/src/components/CreateTemplate/index.tsx [hunk 1]
```diff
@@ -60,6 +60,9 @@ const CreateTemplate = ({
   const { activeOrg } = useContext(GlobalContext);
 
   const [steps, setSteps] = useState(defaultQuestSteps);
+  const refs = useRef([]);
+
+  console.log(refs, "REFS");
```
Comment (andywong418) [inline wondrous-bot-admin/src/components/CreateTemplate/index.tsx:65]: Remove log

File: wondrous-bot-admin/src/pages/quests/create.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ const CreatePage = () => {
     threshold: 1,
   });
 
+
```
Comment (andywong418) [inline wondrous-bot-admin/src/pages/quests/create.tsx:18]: Unnecessary line

File: wondrous-bot-admin/src/pages/quests/index.tsx [hunk 3]
```diff
@@ -38,17 +38,36 @@ const QuestsPage = () => {
...
+          status: null
+        },
+      };
+      setStatuses(null);
+      refetch(variables);
+    }
+  }
+  useEffect(() => {
+    handleFetch()
+  }, [])
```
Comment (andywong418) [inline wondrous-bot-admin/src/pages/quests/index.tsx:69]: this should have activeOrg?.id in the dependency array?

**Comment by Lamperoyge** [comment]
https://github.com/wondrous-dev/wondrous-frontend/pull/1775/files fixed here

**Pull request closed without merging**